plugin/tig_explorer.vim

```diff
@@ -36,7 +36,8 @@ command! TigStatus
       \  call tig_explorer#status()
 
 command! -bang -nargs=* TigOpenFileWithCommit
-      \ call tig_explorer#open_file_with_commit("<bang>",<q-mods>,<f-args>)
+      \ call call('tig_explorer#open_file_with_commit',
+      \            slice(["<bang>",<q-mods>,<f-args>]+['','',''],0,5))
```

Comment:
not working on Vim 8.1 and NVIM v0.4.3

[![Image from Gyazo](https://i.gyazo.com/6be5c3683716348994c4ca0f48b7046c.gif)](https://gyazo.com/6be5c3683716348994c4ca0f48b7046c)

These errors are meaning,

`Unknown function: slice`
`Invalid argument: call`